### PR TITLE
Created quality_control.yml github workflow

### PR DESCRIPTION
**Pull request by WillBanks1**
- Set up quality control workflow
 - Runs pytest on all .py files
 - Runs pylint (set fail threshold to a score of 8).

### CODE REVIEW

File: .github/workflows/quality_control.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# This a workflow to ensure all written project code passes pylint and pytest
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '**.py'
```
Comment (laUmar123) [inline .github/workflows/quality_control.yml:10]: `I believe this should **/*.py`

File: .github/workflows/quality_control.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        run: |
+          pip3 install -r requirements.txt
+          echo "library installation complete"
+      - name: "Run Pytest"
+        run: |
+            pytest --only-path="**.py"
+            echo "Ran pytest"
+      - name: "Run Pylint"
+        run: |
+            pylint --fail-under=8 *.py
```
Comment (laUmar123) [inline .github/workflows/quality_control.yml:33]: I believe this line should be `$(find . -name "*.py")`

Comment (WillBanks1) [reply .github/workflows/quality_control.yml]: Changed this line

**Review by laUmar123 — APPROVED** [review]
Looks good, nice work

**Pull request merged**